src/main/resources/db/migration/V5__alter_token_table.sql

```diff
@@ -0,0 +1,3 @@
+alter table [token]
```

Comment:
Depending on what pr gets merged first we will need to modify the V5 on one of the folder